sympy/solvers/ode/tests/test_ode.py

```diff
@@ -298,6 +298,11 @@ def test_classify_ode():
         ('1st_power_series', 'lie_group')
     assert isinstance(classify_ode(Eq(f(x), 5), f(x), dict=True), dict)
 
+    #This is for new behavior of classify_ode when called internally with default, It should
+    # return the first hint which matches therefore, 'ordered_hints' key will not be there.
+    assert classify_ode(Eq(f(x).diff(x), 0), f(x),dict=True).get('ordered_hints') == None
+    assert classify_ode(Eq(f(x).diff(x), 0), f(x),dict=True).get('default') == 'nth_algebraic'
```

Comment:
There should be a space after a comma.

Why doesn't this assert what the actual output of `classify_ode` is?

Also is the behaviour changed in cases where a hint is specified?

Reply:
> Why doesn't this assert what the actual output of `classify_ode` is?

ok, will change it to that.

> Also is the behaviour changed in cases where a hint is specified?

Earlier classify_ode didn't take hint as argument but now it has so if we pass specific hint it will return solver for that and also checks for the series solution

Reply:
Also the output of this test is something like:
```
{'order': 1,
 'nth_algebraic': <sympy.solvers.ode.single.NthAlgebraic at 0x7f94c63cc160>,
 'nth_algebraic_Integral': <sympy.solvers.ode.single.NthAlgebraic at 0x7f94c63cc160>,
 'default': 'nth_algebraic'}
```
so how should we check with assert?